3-platform/shopping-cart/infra/src/main/java/com/_6core/platform/shopping/cart/infra/adapter/driven/persistence/ShoppingCartRepository.java

```diff
@@ -0,0 +1,10 @@
+package com._6core.platform.shopping.cart.infra.adapter.driven.persistence;
+
+import com._6core.platform.shopping.cart.infra.adapter.driven.persistence.entity.ShoppingCartEntity;
+import reactor.core.publisher.Mono;
+
+public interface ShoppingCartRepository {
```

Comment:
pls check this diagram https://github.com/sokimaaa/6Core/blob/dev/1-documentation/class-diagram/general-usecase.puml

and concrete example: https://github.com/sokimaaa/6Core/blob/dev/1-documentation/class-diagram/get-product-usecase.puml

Be sure of difference between Aggregation/Composition/Extension